src/platform/common_activation.cpp

```diff
@@ -2,21 +2,45 @@
 #include "platform_activation.h"
 #include "string_convert.h"
 #include "pal_error.h"
+#include <string>
+#include <dlfcn.h>
 
 namespace xlang::impl
 {
     xlang_pfn_lib_get_activation_factory try_get_activation_func(
```

Comment:
`func`? Prefer to be readable.

Reply:
I don't understand this comment.

Reply:
`try_get_activation_func` - what does `func` mean? Why not use a complete word?

Reply:
That's what @DefaultRyan named it